client/starwhale/api/_impl/service/types.py

```diff
@@ -1,4 +1,10 @@
 from enum import Enum
+from typing import Any
+
+try:
+    import gradio
```

Comment:
move `import gradio` into the function `all_components_are_gradio`?

Reply:
Done